fedmsg_meta_fedora_infrastructure/scm.py

```diff
@@ -33,6 +33,9 @@ class SCMProcessor(BaseProcessor):
 
     def secondary_icon(self, msg, **config):
         if '.git.receive' in msg['topic']:
+            if 'username' in msg['msg']['commit']:
+                username = msg['msg']['commit']['username']
```

Comment:
Just want to confirm, is `msg['msg']['commit']['username']`  the fas_username? or github username? we need fas_username to generate the libravatar url..